www/front_src/src/Resources/Actions/Resource/Downtime/Dialog.tsx

```diff
@@ -47,6 +47,8 @@ import { Resource } from '../../../models';
 import useAclQuery from '../aclQuery';
 import useDateTimePickerAdapter from '../../../useDateTimePickerAdapter';
 
+const maxEndDate = new Date('2100-01-01');
```

Comment:
Why it's fixed at this date ?

Reply:
Because it doesn't really make sense to make it dynamic, 2100 is far enough :)

Reply:
Ok, because if we always made x years more than the one of today it is not better?